packages/bindings/src/gov.rs

```diff
@@ -86,6 +85,48 @@ pub enum GovProposal {
         /// all code ideas that should be removed from cache to free space
         code_ids: Vec<u64>,
     },
+    // ClientUpdateProposal is a governance proposal. If it passes, the substitute
+    // client's latest consensus state is copied over to the subject client. The proposal
+    // handler may fail if the subject and the substitute do not match in client and
+    // chain parameters (with exception to latest height, frozen height, and chain-id).
+    ClientUpdate {
+        // the client identifier for the client to be updated if the proposal passes
+        subject_client_id: String,
+        // the substitute client identifier for the client standing in for the subject
+        // client
+        substitute_client_id: String,
+    },
+    // UpgradeProposal is a gov Content type for initiating an IBC breaking
+    // upgrade.
+    Upgrade {
+        plan: UpgradePlan,
+        // An UpgradedClientState must be provided to perform an IBC breaking upgrade.
+        // This will make the chain commit to the correct upgraded (self) client state
+        // before the upgrade occurs, so that connecting chains can verify that the
+        // new upgraded client is valid by verifying a proof on the previous version
+        // of the chain. This will allow IBC connections to persist smoothly across
+        // planned chain upgrades
+        upgraded_client_state: ProtoAny,
+    },
+}
+
+// Plan specifies information about a planned upgrade and when it should occur.
+#[derive(Serialize, Deserialize, Clone, PartialEq, Eq, PartialOrd, Ord, JsonSchema, Debug)]
+pub struct UpgradePlan {
+    // Sets the name for the upgrade. This name will be used by the upgraded
+    // version of the software to apply any special "on-upgrade" commands during
+    // the first BeginBlock method after the upgrade is applied. It is also used
+    // to detect whether a software version can handle a given upgrade. If no
+    // upgrade handler with this name has been set in the software, it will be
+    // assumed that the software is out-of-date when the upgrade Time or Height is
+    // reached and the software will exit.
+    pub name: String,
+    // The height at which the upgrade must be performed.
+    // Only used if Time is not set.
```

Comment:
What `Time`? Remove this comment if not applicable.

Reply:
fyi: time was deprecated and must not be set

Reply:
Obviously - I completely removed `Time` as it is deprecated, but forgot to get rid in doc-comments.